### PR TITLE
Feature/separate auth

### PR DESCRIPTION
**Pull request by aspasia**
Part of the authentication extraction. @jaromil please have a look

### CODE REVIEW

File: src/freecoin_lib/config.clj [hunk 1]
```diff
@@ -3,8 +3,7 @@
 
 (def env-vars #{:port :host :base-url :secure :debug
                 :client-id :client-secret :email-config
-                :admin-email :ttl-password-recovery
-                :rpc-config})
+                :ttl-password-recovery :rpc-config})
```
Comment (jaromil) [inline src/freecoin_lib/config.clj]: is this ttl- still needed now that its all moved to just-auth?

Comment (aspasia) [reply src/freecoin_lib/config.clj]: Good point, I will create a config in just-auth for this and the email-config which are specific to the auth

**Comment by aspasia** [comment]
@jaromil fixed, can you check and merge now?

**Comment by jaromil** [comment]
cheers!!!

**Pull request merged**